composer.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "automattic/vip-go-skeleton",
+  "description": "Used for setting up development dependencies across the whole repository.",
+  "homepage": "https://github.com/automattic/vip-go-skeleton",
+  "license": "GPL-2.0-or-later",
+  "require": {
+    "php": ">= 7.2"
```

Comment:
Let's examine what this says: That PHP 7.2 is *required* to be able to run `composer install`.

Is that necessary? 

Well let's run `composer why php`:

```sh
➜  vip-go-skeleton git:(phpcs-support) ✗ composer why php
automattic/vip-go-skeleton                      dev-phpcs-support  requires  php (>= 7.2)
automattic/vipwpcs                              0.3.0              requires  php (>=5.4)
dealerdirect/phpcodesniffer-composer-installer  v0.4.4             requires  php (^5.3|^7)
phpcompatibility/php-compatibility              9.0.0              requires  php (>=5.3)
squizlabs/php_codesniffer                       3.3.2              requires  php (>=5.4.0)
wp-coding-standards/wpcs                        1.2.0              requires  php (>=5.3)
```

Other than the skeleton (this repo), the development tools only need a minimum PHP 5.4, but, since we know the contents of this repo will ultimately need to run on PHP 7.2, I think it could be worth keeping the minimum requirement of this repo as PHP >= 7.2.

As an alternative, we _could_ move the PHP requirement into the `require-dev` section, and keep it as PHP 5.4.